### DISCUSSION TITLE
How can I refresh the dictionaries/reload/reset the machine with a stroke?

### DISCUSSION THREAD

**Discussion by user202729** [opensteno/plover#1340] [Q&A and Support] [answered]
(title.)

*looks like I need to (make a command plugin to) call `engine.reset_machine()`.*

**Comment by benoit-pierre** [comment, marked as answer]
Use `{plover:set_config}`.

**Reply by juliatuttle** [reply]
With what arguments?

**Reply by benoit-pierre** [reply]
No arguments necessary.

**Reply by aerickt** [reply]
`{plover:set_config}` will only reload the dictionaries, is this intended?

**Reply by benoit-pierre** [reply]
Yes, there's no way to reset the machine through a stroke because that does not make any sense: if the machine is disconnected, then you won't be able to stroke anything anyway.

**Reply by aerickt** [reply]
It might still be useful as it could be called from the command line and/or bound to a global shortcut.

**Reply by user202729** [reply]
Actually there's a special case where it makes sense – when the input machine is "keyboard" and it's disconnected then connected again.

Then Plover still receives strokes from that keyboard; however the keyboard capturing/grabbing part stopped working (the Plover output is sent together with the raw Qwerty input)
